Automattic/jetpack
# CRM: Resolves 3107 - Emerald top menu

CRM: Resolves Automattic/zero-bs-crm#3107 - Emerald top menu

## Proposed changes:

Old:
![image](https://github.com/Automattic/jetpack/assets/32492176/24654cd0-25b1-44c1-a5a1-92d1219101a4)

Design proposal:
![image](https://github.com/Automattic/jetpack/assets/32492176/8a32b815-7363-4cc4-b048-a3ba39eb6c6a)

New:
![image](https://github.com/Automattic/jetpack/assets/32492176/f85164c1-f3d4-48ba-b973-35db80b4c1f8)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Everything should function as before. Mobile screen sizes are out of scope in this PR. Make sure to build CSS before testing.
